product_identification/models/productid_category.py

```diff
@@ -0,0 +1,8 @@
+from odoo import fields, models
+
+
+class ProductIdCategory(models.Model):
+    _name = "product.product.id_category"
```

Comment:
except an historical error in odoo, there is no "_" in model name.
maybe "product.identification.category" is enought. don't you think ? (and product.identification instead of product.product.id_number)

Reply:
Improved